documentation/dev/docker.rst

```diff
@@ -0,0 +1,142 @@
+.. docker:
+
+Running via Docker
+======================
+
+FlexMeasures can be run via `docker <https://hub.docker.com/repository/docker/flexmeasures/flexmeasures>`_.
+
+Docker is great to save developers from installation trouble, but also for running FlexMeasures inside modern cloud environments in a scalable manner.
+For now, the use case is local development. Using in production is a goal for later.
+
+We also support running all needed parts of a FlexMeasures EMS setup via `docker-compose <https://docs.docker.com/compose/>`_, which is helpful for developers and might inform hosting efforts. 
+
+.. warning:: The dockerization is still under development.
+
+
+The `flexmeasures` image
+-----------------------------------
+
+Getting the image
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use versions we host at Docker Hub, e.g.:
+
+.. code-block:: bash
+
+    docker pull flexmeasures/flexmeasures:latest
+
+
+You can also build the FlexMeasures image yourself, from source:
+
+.. code-block:: bash
+
+    docker build -t flexmeasures/my-version . 
+
+The tag is your choice.
+
+
+Running
+^^^^^^^^^^^
+
+Running the image might work like this:
+
+.. code-block:: bash
+
+    docker run --env SQLALCHEMY_DATABASE_URI=postgresql://user:pass@localhost:5432/dbname --env SECRET_KEY=blabla -d --net=host flexmeasures/flexmeasures
+
+The two minimal environment variables are the database URI and the secret key.
+
+In this example, we connect to a database running on our local computer, so we use the host network (in the docker-compose section below, we use a Docker container for the database, as well).
+
+Browsing ``http://localhost:5000`` should work.
```

Comment:
Timing out for me.

Reply:
I can go into the container and run flexmeasures CLI commands, though.

Reply:
Can you see something interesting in the `docker-compose logs`?

Reply:
At this point I don't even have `docker-compose` installed.

Reply:
Oh indeed, this is the part without compose. Anyway, you might see interesting logs in the terminal, or use `docker logs <container -name>`

Reply:
Thanks for helping me debug. Apparently, `flexmeasures.cfg` can't be found. This file is mentioned only later in the tutorial, and seems to be needed at this point already.

Reply:
This might be just a warning (that there is no config file) but FM can run with the most crucial settings as env vars. I see:

```
WARNING: File /usr/var/flexmeasures-instance/flexmeasures.cfg could not be found! (work dir is /app)
```

I believe the time-out is due to FlexMeasures thinking it's running in production mode (the default), and then requiring to run on ``https``. This default is security-friendly, but not that developer-friendly. 

Try adding `--env FLASK_ENV=development` to the `docker run` command. I believe that worked for me.

Reply:
Maybe the default FLASK_ENV should not be production, but that also seems like a possible security problem.

Reply:
I don't quite understand why this leads to a time-out and not a re-direct to https and then some 4xx error. But this fix helped to solve the time-out locally for me.

Reply:
Thanks for finding this!

Reply:
Thanks for fixing this! Works as described now.